resources/carrier/en/359.txt

```diff
@@ -12,8 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-35987|Vivacom
-35988|M-Tel
-35989|GLOBUL
-35998|GLOBUL
-359999|MAX
+# 359 - Bulgaria
+
+# Source(s):
+# -----------------------
+# CRC: http://crc.bg:8080/dpls/apex/f?p=923:1512:3529024143652469::NO::P1512_ADV,P1512_REGION_CONTROL,P1512_X:1,1,1
+# -----------------------
+
+# Carriers:
+# -----------------------
+# VIVACOM: vivacom.bg - Bulgarian Telecommunication Company EAD [Българска телекомуникационна компания ЕАД]
+# Mtel: mtel.bg - Mobiltel EAD [Мобилтел ЕАД]
+# Telenor: telenor.bg - Telenor Bulgaria EAD [ТЕЛЕНОР БЪЛГАРИЯ ЕАД]
+# Bulsatcom: bulsat.com - Bulsatcom EAD [БУЛСАТКОМ ЕАД]
+# Max: maxtelecom.bg - Max Telecom Ltd. [МАКС ТЕЛЕКОМ ООД]
+# bob: bob.bg - Mobiltel EAD [Мобилтел ЕАД]
+# -----------------------
+
+# -----------------------
+# Last checked: 7/12/2017
+# Last updated: 7/12/2017
+# -----------------------
+
```

Comment:
Please remove all the comments i.e Source, carriers and last checked/updated. Just mention below comment

"Prefix 988 is assigned to Bob which is a subsidiary of Mtel."